includes/sanitizers/class-amp-video-sanitizer.php

```diff
@@ -305,6 +308,10 @@ private function filter_attributes( $attributes ) {
 			}
 		}
 
+		if ( isset( $out['autoplay'] ) ) {
```

Comment:
A comment would be useful here to provide context. Also, we would only need to remove the `muted` attribute when it is set, so `$out['muted']` could be added as a condition here as well.

```suggestion
		// The amp-video will forcibly be muted whenever it is set to autoplay, so omit the
		// `muted` attribute if it exists.
		if ( isset( $out['autoplay'], $out['muted'] ) ) {
```